### PR TITLE
Add file utilities uid and gid

### PR DESCRIPTION
**Pull request by lydia-duncan**
Though getUID and getGID weren't originally part of the FileSystem proposal,
it seemed reasonable to add them (so that they can be used in copy as well).
Adds a simple test that both work as expected (i.e. the python grab of the
values for a file match our values).

### CODE REVIEW

File: modules/standard/FileSystem.chpl [hunk 1]
```diff
@@ -173,6 +173,54 @@ proc exists(name: string): bool {
   return ret;
 }
 
+/* Returns the user id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
```
Comment (daviditen) [inline modules/standard/FileSystem.chpl]: Typo: Returns _any_ errors... This typo is copied to all four new header comments.

Comment (lydia-duncan) [inline modules/standard/FileSystem.chpl]: Whoops, good catch!

File: modules/standard/FileSystem.chpl [hunk 2]
```diff
@@ -173,6 +173,54 @@ proc exists(name: string): bool {
...
+proc getUID(out err: syserr, name: string): int {
+  extern proc chpl_fs_get_uid(ref result: c_int, filename: c_string): syserr;
+
+  var result: c_int;
+  err = chpl_fs_get_uid(result, name.c_str());
+  return result;
+}
+
+/* Returns the user id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
```
Comment (daviditen) [inline modules/standard/FileSystem.chpl]: This one does not return errors via an out parameter.  Same with the second getGID function.

File: modules/standard/FileSystem.chpl [hunk 3]
```diff
@@ -173,6 +173,54 @@ proc exists(name: string): bool {
...
+  if err != ENOERR then ioerror(err, "in getUID");
+  return ret;
+}
+
+/* Returns the group id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
+   err: a syserr used to indicate if an error occurred
+   name: a string used to indicate the file in question
+*/
+proc getGID(out err: syserr, name: string): int {
```
Comment (daviditen) [inline modules/standard/FileSystem.chpl]: getGID should be before getUID to maintain alphabetical order in the file.

**Comment by daviditen** [comment]
Other than my three little nit-picks, I think it looks good. I think the names you chose are fine.

**Pull request merged**